<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventAppletColorConfig</class>
 <widget class="QWidget" name="EventAppletColorConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="timeBox">
     <property name="title">
      <string>Times</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Urgency for normal events:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KIntSpinBox" name="urgencyBox">
        <property name="toolTip">
         <string>The time left until an event is marked as soon beginning.</string>
        </property>
        <property name="suffix">
         <string>min</string>
        </property>
        <property name="maximum">
         <number>280</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>397</width>
          <height>58</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Urgency for birthdays:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KIntSpinBox" name="birthdayUrgencyBox">
        <property name="toolTip">
         <string>The days left until a birthday or anniversary is marked as being soon.</string>
        </property>
        <property name="suffix">
         <string>days</string>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="normalEventsBox">
     <property name="title">
      <string>Colors for normal events</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Backgroundcolor for near events:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KColorButton" name="urgentColorButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>The backgroundcolor for soon beginning events.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="KIntSpinBox" name="urgentOpacity">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Textcolor for started or passed events:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KColorButton" name="startedColorButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>The textcolor for already started or passed events.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Colors for todos</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Backgroundcolor for todos:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KColorButton" name="todoColorButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>The backgroundcolor for common todos.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="KIntSpinBox" name="todoOpacity">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="showFinishedTodos">
        <property name="toolTip">
         <string>Show or hide already finished todos.</string>
        </property>
        <property name="text">
         <string>Show finished todos</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>371</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Background for finished todos:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KColorButton" name="finishedTodoButton">
        <property name="toolTip">
         <string>The backgroundcolor for already finished todos.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="KIntSpinBox" name="finishedTodoOpacity">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>KOrganizer colors</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KIntSpinBox" name="korganizerOpacity">
        <property name="suffix">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>93</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
